.github/workflows/sweeper.yaml

```diff
@@ -25,6 +25,12 @@ jobs:
           go-version-file: test/hack/resource/go.mod
           check-latest: true
           cache-dependency-path: "test/hack/resource/go.sum"
-      - run: go run main.go
+      # eu-north-1 contains the soak tests and resources should be sweept if they are older then 8 days
+      - run: |
+          if [[ "${{ matrix.region }}" == "eu-north-1" ]]; then
```

Comment:
Is it possible for users to run other tests in eu-north-1? If so, does this mean that the resources take a lot longer to get swept up? Is there a way we can make sure this doesn't happen?